Connnect/JoinViewController.swift

```diff
@@ -16,6 +16,9 @@ class JoinViewController: UIViewController {
     @IBAction func newChannel(sender: UIButton) {
         guard let newChannelName = joinTextField.text else { return }
 
+        guard newChannelName != ""
+            else { return }
+        
```

Comment:
It's pretty clear in use that if you try to create a new channel without a name. Then the button does not do anything

We could do an observer on the text box and only enable the button once they have entered some text.